Application/EdFi.Ods.Api.IntegrationTestHarness/Modules/UpdateAdminDatabaseModule.cs

```diff
@@ -0,0 +1,28 @@
+﻿// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using Autofac;
+using EdFi.Ods.Api.Common.Configuration;
+using EdFi.Ods.Api.Common.Constants;
+using EdFi.Ods.Api.Common.Container;
+using EdFi.Ods.Api.Common.ExternalTasks;
+
+namespace EdFi.Ods.Api.IntegrationTestHarness.Modules
+{
+    public class UpdateAdminDatabaseModule : ConditionalModule
+    {
+        public UpdateAdminDatabaseModule(ApiSettings apiSettings)
+            : base(apiSettings, nameof(UpdateAdminDatabaseModule)) { }
+
+        public override bool IsSelected() => IsFeatureEnabled(ApiFeature.UpdateAdminDatabase);
+
+        public override void ApplyConfigurationSpecificRegistrations(ContainerBuilder builder)
+        {
+            builder.RegisterType<UpdateAdminDatabase>()
+                .As<IExternalTask>()
+                .SingleInstance();
+        }
+    }
```

Comment:
We do not need to add flag here, this is a required process so just implement Module, and remove the config flag